### DISCUSSION TITLE
Verifying you are a student

### DISCUSSION THREAD

**Discussion by spadefuljacknoir** [community/community#50937] [GitHub Education]
I have sent GitHub Campus my physical copy of my university ID, but they don't give me my discount anyway. What can I do?
(I am from Ukraine)

**Comment by spadefuljacknoir** [comment]
I did it multiple times and from different angles, it doesn't do the trick anyway.
I also added my bio in my account in my native language and all the info from the ID too.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @spadefuljacknoir, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by spadefuljacknoir** [reply]
thanks

**Comment by samus-aran** [comment]
Hi @spadefuljacknoir  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
